### DISCUSSION TITLE
Chrome sets payee label to "Google store" in the SPC display

### DISCUSSION THREAD

**Discussion by apouillard** [w3c/secure-payment-confirmation#258] [Q&A] [answered]
Hi SPC developers !

Could you please explain why Chrome shows "Google Store" in the payee label of SPC window ? 
I use version 116.0.5845.97 of Chrome and I don't know from which version it has been set to this value, however I can remember a time where the value was simply "Store"...

Thanks for the explanation

@stephenmcgruer 
@rsolomakhin 
@ianbjacobs

**Comment by stephenmcgruer** [comment]
Hi @apouillard ,

Can you share a reproduction page and/or screencast of the problem? It's unclear to me whether you're referring to the label or the value for the payeeName + payeeOrigin:

![image](https://github.com/w3c/secure-payment-confirmation/assets/153264/b3615f25-6f20-4af2-9c9a-8b3bbe442e9a)

As in this screenshot, I don't see the problem mentioned on our test page - the label is 'Store' and the value is correctly the specified payeeName + payeeOrigin.

**Comment by apouillard** [comment]
Hi @stephenmcgruer 

Here is a fresh screenshot 

![image](https://github.com/w3c/secure-payment-confirmation/assets/36415388/144bf928-a7f4-4b27-bcc7-cb3b168a8f66)

Could this behaviour be linked with some spc or experimental flags enabled in my settings ?

**Comment by stephenmcgruer** [comment, marked as answer]
Ah, this appears to be a translation bug in Chrome! Thank you for the report, I have opened https://bugs.chromium.org/p/chromium/issues/detail?id=1473949 to track this, as it is a browser bug rather than a spec bug.

**Reply by apouillard** [reply]
great, thanks Stephen
